src/libsyncengine/jobs/network/abstractnetworkjob.cpp

```diff
@@ -379,7 +379,7 @@ bool AbstractNetworkJob::sendRequest(const Poco::URI &uri) {
         }
 
         if (isProgressTracked()) {
-            _progress += itEnd - itBegin;
+            setProgress(getProgress() + itEnd - itBegin);
```

Comment:
Idea for later : implement a method `addProgress` or something like that to avoid calling `getProgress()`

Reply:
quickwin 😉 [5e7034a](https://github.com/Infomaniak/desktop-kDrive/pull/139/commits/5e7034a0e890df417e4ff87b9be22fbe02ec8018)